schemas/com.vimeo/interactive_overlay_panel_click_event/1-0-0

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a Vimeo interactive overlay panel click",
```

Comment:
Also not sure what is the difference of this one compared to the `interaction` schema? Is it because this one is "overlay"?

Reply:
Clarified [here](https://github.com/snowplow/iglu-central/pull/1282#discussion_r1219939228). This is the "event" schema for the `interaction` context.